guides/release/reference/accessibility-guide.md

```diff
@@ -0,0 +1,109 @@
+The purpose of this guide is to give every Ember developer a starting point for making your Ember application minimally accessible. As accessibility can be a complex issue, it should be noted that this guide only covers a some of the basics, and should not be considered a comprehensive guide. A good place to start is [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/) provided by the Web Accessibility Initiative (WAI).
+
+## Ember Application Configuration
+
+### Optional Feature: Application Template Wrapper
+
+If you are using the [application template wrapper](#toc_optional-features) enabled (default state), then you will need to add certain aria roles to your [landmark regions](https://www.w3.org/WAI/PF/aria/roles#landmark_roles), even if you are using native HTML elements, because those regions are not the direct child descendant of the body element (they are the children of the div that wraps the Ember app).
+
+If you disable the [application template wrapper](#toc_optional-features), you will not need to add role attributes to your landmark regions when they are the direct descendant of the body element, and they are using native HTML elements. This is the preferred approach for accessible applications. 
+
+**Application Template Wrapper Disabled** _(preferred)_
+```html
+<body>
+  <header></header>
+  <main></main>
+  <footer></footer>
+</body>
+```
+
+**Application Template Wrapper Enabled**
+```html
+<body>
+  <div class="ember-view">
+    <header role="banner"></header>
+    <main role="main"></main>
+    <footer role="contentinfo"></footer>
+  </div>
+</body>
+```
+
+> Note: To learn more about landmark roles and how to use them: [https://www.w3.org/WAI/PF/aria/roles#landmark_roles](https://www.w3.org/WAI/PF/aria/roles#landmark_roles). Still need more help? Visit the #topic-a11y channel in [Ember chat](https://emberjs.com/community/). 
```

Comment:
For some reason this "blockquote" is causing some rendering issues 🤔 https://deploy-preview-462--ember-guides.netlify.com/release/reference/accessibility-guide/#toc_ember-application-vs-roleapplication

The question is: do we need to solve the rendering issues or can we do it as a "callout" with the tomster/zoey for now?

Reply:
how would i do it as a callout?

Reply:
Here is an example from https://github.com/ember-learn/guides-source/blob/master/guides/release/components/defining-a-component.md#L38

```html
<div class="cta">
  <div class="cta-note">
    <div class="cta-note-body">
      <div class="cta-note-heading">Zoey says...</div>
      <div class="cta-note-message">
In Ember templates there are different ways to invoke a Component. The syntax above is referred to as angle bracket invocation syntax, and it might not look familiar if you are looking at older code samples that use the classic invocation syntax. For more examples of ways to use Components in a template, see the <a href="../../reference/syntax-conversion-guide">Syntax Conversion Guide</a>, a <a href="https://guides.emberjs.com/v3.6.0/components/defining-a-component/">previous version of the Guides</a> or <a href="https://emberjs.com/api/ember/3.6/classes/Component">Ember.js API documentation</a>.
      </div>
    </div>
    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
  </div>
</div>
```

The thing you have to be careful of is that the message does not render any markdown so you need to the links old school 👍